Compilation Error: Missing Members in DaikinS21Climate (visual_min/max_temperature_override_, visual_target/current_temperature_step_override_)

Hello,

When compiling with ESPHome 2026.1.4, the following errors occur:

```
src/esphome/components/daikin_s21/climate/daikin_s21_climate.cpp:44:27: error: 'class esphome::daikin_s21::DaikinS21Climate' has no member named 'visual_min_temperature_override_'
src/esphome/components/daikin_s21/climate/daikin_s21_climate.cpp:45:52: error: 'class esphome::daikin_s21::DaikinS21Climate' has no member named 'visual_min_temperature_override_'
...
```

These errors concern the following members:
- `visual_min_temperature_override_`
- `visual_max_temperature_override_`
- `visual_target_temperature_step_override_`
- `visual_current_temperature_step_override_`

These variables are referenced in the source code but are not declared in the `DaikinS21Climate` class.

**Configuration used:**
- ESPHome 2026.1.4
- External component: `github://asund/esphome-daikin-s21@main`

Could you check if these members should be added or if their usage should be removed?

Thank you.

--- a/components/daikin_s21/climate/daikin_s21_climate.cpp
+++ b/components/daikin_s21/climate/daikin_s21_climate.cpp
@@ -41,24 +41,36 @@ void DaikinS21Climate::setup() {
   this->heat_params.target_pref = global_preferences->make_preference<int16_t>(h + 3);
   // populate default traits
   this->traits_.add_feature_flags(climate::CLIMATE_SUPPORTS_CURRENT_TEMPERATURE | climate::CLIMATE_SUPPORTS_ACTION);
+#ifdef USE_CLIMATE_VISUAL_OVERRIDES
   if (std::isfinite(this->visual_min_temperature_override_)) {
     this->traits_.set_visual_min_temperature(this->visual_min_temperature_override_);
-  } else {
+  } else
+#endif
+  {
     this->traits_.set_visual_min_temperature(std::min({this->heat_cool_params.min, this->cool_params.min, this->heat_params.min}).f_degc());
   }
+#ifdef USE_CLIMATE_VISUAL_OVERRIDES
   if (std::isfinite(this->visual_max_temperature_override_)) {
     this->traits_.set_visual_max_temperature(this->visual_max_temperature_override_);
-  } else {
+  } else
+#endif
+  {
     this->traits_.set_visual_max_temperature(std::max({this->heat_cool_params.max, this->cool_params.max, this->heat_params.max}).f_degc());
   }
+#ifdef USE_CLIMATE_VISUAL_OVERRIDES
   if (std::isfinite(this->visual_target_temperature_step_override_)) {
     this->traits_.set_visual_target_temperature_step(this->visual_target_temperature_step_override_);
-  } else {
+  } else
+#endif
+  {
     this->traits_.set_visual_target_temperature_step(SETPOINT_STEP.f_degc());
   }
+#ifdef USE_CLIMATE_VISUAL_OVERRIDES
   if (std::isfinite(this->visual_current_temperature_step_override_)) {
     this->traits_.set_visual_current_temperature_step(this->visual_current_temperature_step_override_);
-  } else {
+  } else
+#endif
+  {
     this->traits_.set_visual_current_temperature_step(TEMPERATURE_STEP.f_degc());
   }
   this->traits_.set_supported_fan_modes({climate::CLIMATE_FAN_AUTO, climate::CLIMATE_FAN_QUIET});

